help/en/html/tools/scripting/HowTo.shtml

```diff
@@ -160,8 +160,58 @@
 		these purposes if there is the possibility that you will be changing your control system in which case 
 		system names will be changing as well.</p>
 		
-		<p>Click for more information on <a href="help/en/html/doc/Technical/Names.shtml"> Names and Naming</a>.</p>
+		<p>Click for more information on <a href="http://jmri.org/help/en/html/doc/Technical/Names.shtml"> Names and Naming</a>.</p>
```

Comment:
You need to remove `http://jmri.org/` from here. Otherwise the local copy of this link included in JMRI will not be used in the JMRI help browser.